TestProject/tests/cellularAutomat/BoardTest.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Tests\CellularAutomat;
+
+use GameOfLife\cellularAutomat\Board;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests whether \CellularAutomat\Board works as expected.
+ */
+class BoardTest extends TestCase
+{
+    protected $board;
+
+    protected $width = 10;
+    protected $height = 10;
+    protected $x = 5;
+    protected $y = 5;
+
+    /**
+     * Sets height and width of the board.
+     *
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        $this->board = new Board(10, 10);
+    }
+
+    /**
+     * Initialize an empty board -> cells are all set on 0.
+     *
+     * @param $_board
+     * @return bool
+     */
+    private function emptyBoard($_board)
+    {
+        $boardIsEmpty = true;
+        foreach ($_board as $row)
+        {
+            foreach ($row as $cell)
+            {
+                if ($cell != 0)
+                    $boardIsEmpty = false;
+            }
+        }
+        return $boardIsEmpty;
+    }
+
+    /**
+     * Asserts that width has the same value that was defined inside the constructor.
+     *
+     * @return void
+     * @test
+     */
+    public function checksConstructorWidthIsActuallyWidth()
+    {
+        $this->assertEquals(10, $this->board->getWidth());
+    }
+
+    /**
+     * Asserts that height has the same value that was defined inside the constructor.
+     *
+     * @return void
+     * @test
+     */
+    public function checksIfConstructorHeightIsActuallyHeight()
+    {
+        $this->assertEquals(10, $this->board->getHeight());
+    }
+
+    /**
+     * Asserts that the created board  is empty.
+     *
+     * @test
+     */
+    public function boardIsEmpty()
+    {
+        $board = $this->board->getBoard();
+        $boardIsEmpty = $this->emptyBoard($board);
+
+        $this->assertTrue($boardIsEmpty);
+    }
+
+    /**
+     * Asserts that the created board is not empty.
+     *
+     * @test
+     */
+    public function setCellAssertBoardIsNotEmpty()
+    {
+        $this->board->setCell(0,0,1);
+        $board = $this->board->getBoard();
```

Comment:
This line is a perfect example how a bad method naming leads to unreadable code.
Looking from the outside another developer does not know what the returnvalue of getBoard() really is.
Would'nt that look so much nicer and would be understood better?
```php
 $boardArray = $this->board->getBoardAsArray(); 
```
I leave it to you if you want to change that or not. If you do, use the phpStorm refactoring support to save you a ton of manual changes.